server/test/flashcard_test.js

```diff
@@ -0,0 +1,325 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import app from "../index.js";
+import Flashcard from "../models/flashcard.js"
+
```

Comment:
in your branch, it seems that you're `package.json` is not the latest. can you send us your current `package.json` on discord pls? does this test still run with the latest `package.json`?  I'm curious to know since I would like to also do it your way (`"type": "module"`) if possible

Reply:
{
  "name": "server",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "type": "module",
  "scripts": {
    "start": "node index.js",
    "dev": "node index.js",
    "test": "mocha --timeout 5000"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "bcrypt": "^5.0.1",
    "bcryptjs": "^2.4.3",
    "body-parser": "^1.20.0",
    "cors": "^2.8.5",
    "dotenv": "^16.0.1",
    "express": "^4.17.3",
    "jsonwebtoken": "^8.5.1",
    "mongoose": "^6.3.0"
  },
  "devDependencies": {
    "chai": "^4.3.6",
    "chai-http": "^4.3.0",
    "mocha": "^10.0.0",
    "nodemon": "^2.0.16",
    "prettier": "2.6.2"
  }
}

Reply:
ok, go ahead and implement `require()` then